manifests/g/GoHarbor/Harbor/0.0.1/GoHarbor.Harbor.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: GoHarbor.Harbor
+PackageVersion: 0.0.1
+PackageLocale: en-US
+Publisher: Harbor
+PackageName: Harbor CLI
+PackageURL: https://goharbor.io/
+License: Apache License 2.0
+LicenseUrl: https://github.com/goharbor/harbor-cli/blob/main/LICENSE
+ShortDescription: [Sandbox] Official Harbor CLI

```

```suggestion
ShortDescription: \[Sandbox\] Official Harbor CLI
```

It seems that square brackets are reserved characters in YAML.